tasks/main.yml

```diff
@@ -45,6 +45,7 @@
     - named.conf.options
   notify:
     - reload bind9
+  register: named_conf
```

Comment:
Why do you register this variable?

Reply:
As explained in the PR text, I need to check it outside the role for the interaction between two roles which act on bind9, [here](https://framagit.org/ulvida/belvil.net/blob/master/roles/alternc_bind9/tasks/main.yml#L4) in [this set of playbooks](https://framagit.org/ulvida/belvil.net/). Maybe we can put a comment.

Reply:
Alright, please add the comment, than we're good to go.

Reply:
I think you should rename the variable to something in the bind9 namespace. Then use your playbook to assign the variable `named_conf = bind9_....`

Reply:
> I think you should rename the variable to something in the bind9 namespace.

you're right. done, with a comment. 

> Then use your playbook to assign the variable `named_conf = bind9_....`
I can also change my conde outside the role. 

BTW: maybe it's good to manage also a bind9 namespace for handlers names: they can be called outside the role, and ansible doesn't seem to manage any kind of precedence between handlers with the same name.